static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -10,9 +10,7 @@ import type {
   CourseRun,
   FinancialAidUserInfo
 } from "../../../flow/programTypes"
-import { makeCouponMessage } from "../../../lib/coupon"
```

Comment:
I think we should remove this function from `lib/coupon.js` as well.

Reply:
Yes, we can remove it. But I think It would be better to include that in removing the coupons completely. There is a lot refactoring that is involved so I am trying to avoid it for now.